src/app.constants.ts

```diff
@@ -3,7 +3,9 @@ import { UserJourney } from "./utils/state-machine";
 export const PATH_DATA: {
   [key: string]: { url: string; event?: string; type?: UserJourney };
 } = {
+  GOVUK_HOMEPAGE: { url: "https://www.gov.uk" },
```

Comment:
Setting a variable is the right thing to do, but I think this is the wrong place to put it. `PATH_DATA` is all the internal paths and is used by the state machine to work out where someone's allowed to go next. 

I think it'd be better to put a `getManageGovUkEmailsUrl()` to [config.ts](https://github.com/alphagov/di-authentication-account-management/blob/main/src/config.ts) and bring that into your controllers.

Reply:
Good point @alex9smith, thanks for explaining the difference